workbench/runtime.py

```diff
@@ -238,6 +241,8 @@ class WorkbenchRuntime(Runtime):
     `self.runtime`.
 
     """
+    anonymous_student_id = '7118f8d6-11c3-11e9-a324-7f11806df4d3'
+    hostname = '127.0.0.1:8000'
```

Comment:
There should probably be a comment explaining why these are here, otherwise they're likely to be deleted during code cleanup.  Also, does `anonymous_student_id` need to be this particular value or can it be an arbitrary UUID generated during startup?

Reply:
I'll keep it a dummy static value, but make it clearer. 

I also added tests to those params, so it won't be cleaned up by mistake.